pkg/cloudprovider/types.go

```diff
@@ -48,19 +49,15 @@ type Capacity interface {
 	Validate(context.Context) error
 }
 
-// Constraints for an efficient binpacking solution of pods onto nodes, given
-// overhead and node constraints.
-type Constraints struct {
-	v1alpha1.Constraints
-	// Pods is a list of equivalently schedulable pods to be binpacked.
-	Pods []*v1.Pod
-	// Overhead resources per node from daemonsets.
-	Overhead v1.ResourceList
+// Packing TODO
```

Comment:
What is the todo here?

Reply:
A reminder for me to write a nice comment here :)